pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -478,6 +478,7 @@ let
 
       BTRFS_FS_POSIX_ACL = yes;
 
+      # FIXME: Make these by default 'yes' when LTS kernel (default) is at least version 6.7
```

Comment:
We also have older LTS releases. So we would only drop this when the last kernel older than 6.7 is gone.

Reply:
I think we can drop the comment. The `whenAtLeast` makes it clear when we can get rid of the condition.
 
```suggestion
```